testplan/testing/multitest/base.py

```diff
@@ -572,9 +572,19 @@ def get_tags_index(self):
             )
         return self._tags_index
 
-    def skip_step(self, step):
+    def skip_step(self, step) -> bool:
         """Check if a step should be skipped."""
-        if step in (
+        if step == self._run_error_handler:
+            if (
+                self.resources.start_exceptions
+                or self.resources.stop_exceptions
+                or self._get_error_logs()
+            ):
+                return False
+            else:
+                self.logger.critical("Skipping error handler")
```

Comment:
actually will this line being triggered in most cases? (testing passed or error handler not present)

Reply:
You are right, log has been removed!